api/src/test/java/org/openmrs/module/fhir2/api/search/ConditionSearchQueryTest.java

```diff
@@ -649,12 +645,15 @@ public void searchForConditions_shouldReturnConditionByUnboundedOnsetDate() {
 	
 	@Test
 	public void searchForConditions_shouldReturnConditionByOnsetAgeLessThanHour() {
```

Comment:
These changes allowed me to remove the mock for the MockedCalendarFactoryConfiguration